src/hacker.rs

```diff
@@ -1,5 +1,6 @@
 //! The hacker prelude, a fully 64-bit environment for audio processing.
 
+pub use super::adsr::*;
```

Comment:
`adsr_live`, `adsr_fixed`, and `adsr` should have a corresponding function in the preludes that takes no generics.

Reply:
Ok, makes sense. I'll work this out once we figure out what to do about the `AtomicCell` issue.

Reply:
See #19.